Config.psm1

```diff
@@ -112,6 +112,11 @@ function Get-AvailableConfigOptions {
 }
 
 function Get-WindowsImageConfig {
+     <#
+    .SYNOPSIS
+     This function generates a dictionary of key:value pairs with the config options found
```

Comment:
This function reads the ini file given as a parameter and returns a dictionary of config options for the Windows image to be generated. If there are no values for a set of keys defined in Get-AvailableConfigOptions, the default values will be used instead.